### PR TITLE
feat: rearrange order of tabs

### PR DESCRIPTION
**Pull request by dergigi**
Maps to what is probably the most common user flow:

```
[ Receive -> Jam / Earn -> Send ]
```

Screenshot 2022-05-11 at 16 37 10

receive-jam-earn-send

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -36,11 +36,29 @@ const WalletPreview = ({ wallet, walletInfo, unit, showBalance }) => {
...
   const { t } = useTranslation()
+  const serviceInfo = useServiceInfo()
 
   const scheduleEnabled = isFeatureEnabled('schedule')
+  const [collaborativeOperationRunning, setCollaborativeOperationRunning] = useState(false)
+
+  useEffect(() => {
+    const coinjoinInProgress = serviceInfo && serviceInfo.coinjoinInProgress
+    setCollaborativeOperationRunning(coinjoinInProgress)
+  }, [serviceInfo])
```
Comment (dnlggr) [inline src/components/Navbar.jsx:47]: Would it work to just pass in `coinjoinInProgress` as a prop to the `CenterNav` component similar to what we do with `makerRunning`?

**Review by dnlggr — APPROVED** [review]
👌 GTNM 💃 

Makes sense!

**Comment by editwentyone** [comment]
I updated the cheatsheet to represent this order. also shorten the schedule description and finished the list with a checkmark to underline, that the list ends here. 

https://www.figma.com/file/kfejZJFlwBywvLEnPEmJo1/?node-id=2850%3A67638

image

**Comment by dnlggr** [comment]
> I updated the cheatsheet to represent this order. also shorten the schedule description and finished the list with a checkmark to underline, that the list ends here.
> 
> https://www.figma.com/file/kfejZJFlwBywvLEnPEmJo1/?node-id=2850%3A67638
> 
> image

Nice, can you put this into an issue for v0.0.6 so we can keep track of it? 🙏

**Pull request merged**